requestor/server.py

```diff
@@ -16,15 +17,64 @@
 ERIGON_CLS = getattr(erigon_services, os.environ.get('ERIGON_CLASS', 'Erigon'))
 
 
+class ErigonServiceWrapper(ServiceWrapper):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._name = None
+        self._created_at = datetime.utcnow()
+        self._stopped_at = None
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name
```

Comment:
What is the advantage of making it a property when it can be a public field?

Reply:
There is none.
This was left this way after some earlier version.
Fixed in b8be0e6.